code/core/src/manager/settings/Checklist/checklistData.tsx

```diff
@@ -0,0 +1,173 @@
+import React from 'react';
+
+import { Link } from 'storybook/internal/components';
+
+import type { ChecklistData } from './Checklist';
+
+export const checklistData: ChecklistData = {
+  sections: [
+    {
+      id: 'build',
+      title: 'Build',
+      items: [
+        {
+          id: 'whats-new-sb-9',
+          label: "See what's new",
+          start: ({ api }) => api.navigate('/settings/whats-new'),
+        },
+        {
+          id: 'add-component',
+          label: 'Add component',
+          predicate: ({ complete }) => complete(),
+        },
+        {
+          id: 'add-5-10-components',
+          label: 'Add 5-10 total components',
+          predicate: ({ complete }) => complete(),
+        },
+        {
+          id: 'check-improve-coverage',
+          label: 'Check + improve coverage',
+          predicate: ({ complete }) => setTimeout(complete, 3000),
+        },
+      ],
+    },
+    {
+      id: 'test',
+      title: 'Test',
+      items: [
+        {
+          id: 'run-tests',
+          label: 'Run tests',
+          predicate: ({ complete }) => complete(),
```

Comment:
`predicate` is a bit vague. When is this callback function actually executed, and what is its purpose?